simple/util/dc_client.py

```diff
@@ -79,13 +88,39 @@ def resolve_place_entities(
     entities: list[str],
     entity_type: str = None,
     property_name: str = "description") -> dict[str, str]:
+  return asyncio.run(
+      resolve_place_entities_async(entities, entity_type, property_name))
+
+
+async def resolve_place_entities_async(
+    entities: list[str],
+    entity_type: str = None,
+    property_name: str = "description") -> dict[str, str]:
+
+  chunks = chunked(entities, _RESOLVE_BATCH_SIZE)
+
+  resolved: dict[str, str] = {}
+  async with AsyncClient(limits=_HTTPX_LIMITS) as client:
+    futures: dict[str, str] = [
+        _resolve_place_entities_chunk(client, chunk, entity_type, property_name)
+        for chunk in chunks
+    ]
+    for resolved_chunk in await asyncio.gather(*futures):
```

Comment:
looks like now if one request failed, all chunks are canceled? Is there a per request retry mechanism easily available from asyncio library?

Reply:
There is a callback mechanism which we can use. Let me also investigate if there is an easier option for retries. Will raise a separate PR for it.